pkg/meta/base.go

```diff
@@ -274,7 +274,7 @@ func (m *baseMeta) cleanupDeletedFiles() {
 		if ok, err := m.en.setIfSmall("lastCleanupFiles", time.Now().Unix(), 60); err != nil {
 			logger.Warnf("checking counter lastCleanupFiles: %s", err)
 		} else if ok {
-			files, err := m.en.doFindDeletedFiles(time.Now().Add(-time.Hour).Unix())
+			files, err := m.en.doFindDeletedFiles(time.Now().Add(-time.Hour).Unix(), 1000)
```

Comment:
100000

Reply:
10k might be enough. 60s * 100 deletes/s = 6k